apps/teacher/src/app/documents/[documentId]/page.tsx

```
@@ -110,6 +119,20 @@ function StatusModal({
           ))}
         </div>
 
+        {needsReason && (
+          <div className="fade-in slide-in-from-top-2 mb-8 animate-in">
+            <label className="mb-2 block font-medium text-[15px] text-gray-600">
+              {selectedStatus} 사유
+            </label>
```

Biome (2.4.15)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)